node/flows.re

```diff
@@ -14,6 +14,7 @@ type ignore = [
   | `Not_current_block_producer
   | `Pending_blocks
 ];
+type endpoint_errors('a) = [> | `Failed_to_verify_payload] as 'a;
```

Comment:
This type is not needed or being used on something meaningful, please remove it

Reply:
Okay. Removing it.